templates/zerver/help/add-a-bot-or-integration.md

```diff
@@ -0,0 +1,39 @@
+# Add a Bot or Integration
+
+Bots and integrations are features that are accesible to all members of an organization.
+
+## To Add a Bot
+
+{!go-to-the.md!} [Your Bots](/#settings/your-bots)
+{!settings.md!}
+
+2. On this page, under the **Add a Bot** window, you can enter a name, an email, and
+optionally an avatar for your bot. Then click the **Create Bot** button to create your bot.
+
+    ![Bots page](/static/images/help/add_a_new_bot.png)
+
+3. An entry such as the following should appear above the **Add a New Bot** view, confirming
+the creation of your bot.
+
+    ![Example bot](/static/images/help/bot_example.png)
+
+4. You have succesfully created your bot. You now have access to your bot's API key and API
+configuration file (`.zuliprc`) which you will need for integrations that you would like
+to use with this bot. You can click on the green download icon, from the previous image,
```

Comment:
I've called it the download icon, and a downward pointing arrow is almost the universal symbol for downloading. Shouldn't that be enough?